### PR TITLE
configure emailjs and set state for template email

### PR DESCRIPTION
**Pull request by Guy-deMessimy**

### CODE REVIEW

**Review by AmjadLille — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by AmjadLille — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -57,6 +58,5 @@
   "devDependencies": {
     "concurrently": "^5.3.0",
     "prettier": "^2.2.1"
-  },
-  "proxy": "http://localhost:5000"
+  }
```
Comment (LoicBrassart) [inline package.json:61]: You make me doubt myself...
This change seems to me it will break everything (add only one bracket, without it's counterpart)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
I think you made a mistake, please check it

**Review by LoicBrassart — APPROVED** [review]
Nevermind, my mistake!

**Pull request merged**